docusaurus/docs/index.mdx

```diff
@@ -46,10 +46,19 @@ If you have questions, feel free to [contact us](https://knapsackpro.com/contact
 
 Can't see your language/testing framework above? [Let us know](https://docs.google.com/forms/d/e/1FAIpQLSe7Z6k__VczmRMmXykjA5i2MVEA3nEJ90gbiIeCRjecWhPOig/viewform?hl=en).
 
-You can still use Knapsack Pro by writing your integration on top of the JavaScript SDK or using the Knapsack Pro API:
+## Troubleshooting
```

Comment:
I'd put this at the bottom.

I believe "Can't see your language/testing framework above?" should be close to "Write your integration".